src/shell/Shell.cpp

```diff
@@ -636,8 +647,15 @@ static void executeWAST(Store* store, const std::string& filename, const std::ve
     wabt::WastParseOptions parse_wast_options(features);
     auto result = wabt::ParseWastScript(lexer.get(), &script, &errors, &parse_wast_options);
     if (!wabt::Succeeded(result)) {
-        RELEASE_ASSERT_NOT_REACHED();
-        return;
+        fprintf(stderr, "Syntax error");
+        if (errors.size() > 1)
+            fprintf(stderr, "s");
+        fprintf(stderr, ":\n");
+        int i = 1;
+        for (auto e : errors)
+            fprintf(stderr, "%15d.) %s\n", i++, e.message.c_str());
```

Comment:
It is rare to have many errors, and their index is not improtant, so I think just print the error messages.

Reply:
Okay